README.md

```diff
@@ -52,6 +52,7 @@ Twitter              | Fediverse                           | Name
 @kittylyst           | @kittylyst@mastodon.social          | Ben Evans
 @ludovic_dev         | @ludovic_dev@mastodon.social        | Ludovic Henry
 @lukaseder           | @lukaseder@mastodon.social          | Lukas Eder
+@ithempel            | @sepp@chaos.social                  | Sebastian Hempel
```

Comment:
Can you please do me a favour and keep the alphabetic ordering of Twitter handles? So after icougil if I see correctly. Thx!

Reply:
No problem! Was looking for the ordering but did not recognize, that the list was order after the handle.